### DISCUSSION TITLE
Sleep Mode is not triggered when setting min sunset time

### DISCUSSION THREAD

**Discussion by DarthSonic** [basnijholt/adaptive-lighting#1030] [Q&A] [answered]
I set min sunset time to 23:30:00 and sleep time transition to 1 second, but sleep time is not activating.

Instead, I must activate myself. But then the lights won't adjust until turning off and on again.

I can live with the automation cause then I can set sleep time on weekends later, but how could I trigger lights to update when sleep time is activated by automation?

**Comment by djurny** [comment]
Hi there,
Did you als enable [`transition_until_sleep`](https://github.com/basnijholt/adaptive-lighting?tab=readme-ov-file#memo-options)? If so, the docs are not clear if that will effectively enable the sleep mode, but it should use the sleep mode _settings_ as transition target.

Also, which brand/type of lights do you use? I had some issues with Ikea branded zigbee lights, that would not respond well when they were busy with some transition. For Ikea branded lights, you should experiment with `separate_turn_on_commands` as mentioned [here](https://github.com/basnijholt/adaptive-lighting?tab=readme-ov-file#bulb-bulb-specific-issues).

Groetjes,

**Reply by DarthSonic** [reply, marked as answer]
All lights are Hue. transition was disabled first. meanwhile I enabled it for 30 minutes.

Also I rearranged zha groups and I am using a single adaptive lightning config pet group what solved other issues too. it seems to be important to best control the defined groups in single calls and not mix zha groups into HA groups or similar.
